### PR TITLE
Support multiple operations instances

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description
This PR adds support for multiple instances within the Operations subnet.  It also adds a Terraform output for the remote desktop (Guacamole) URL, as well as some more useful info in the README.

## 💭 Motivation and Context
There was a request to spin up more than one Kali instance in the Operations subnet.  This PR enables that in a flexible way that will support additional types of instances (phishing, Cobalt Strike, Windows, etc.) in the future.

I also took this opportunity to update the README to be a bit more comprehensive and also to add a useful output with the Guacamole URL.

## 🧪 Testing
All pre-commit tests passed and all of these changes were successfully applied in the COOL.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +72,7 @@ Coming soon!
 | cert_bucket_name | The name of the AWS S3 bucket where certificates are stored. | string | `cisa-cool-certificates` | no |
 | cool_domain | The domain where the COOL resources reside (e.g. "cool.cyber.dhs.gov"). | string | `cool.cyber.dhs.gov` | no |
 | guac_connection_setup_path | The full path to the dbinit directory where initialization files must be stored in order to work properly (e.g. "/var/guacamole/dbinit"). | string | `/var/guacamole/dbinit` | no |
+| operations_instance_counts | A map specifying how many instances of each type should be created in the operations subnet (e.g. {\ "kali\": 1 }). | map(number) | `{ "kali": 1 }` | no |
```
Comment (jsf9k) [inline README.md]: A list of possible keys would be good here.  The list can be appended to as more instance types are supported.

File: variables.tf [hunk 2]
```diff
@@ -72,6 +72,12 @@ variable "guac_connection_setup_path" {
   default     = "/var/guacamole/dbinit"
 }
 
+variable "operations_instance_counts" {
+  type        = map(number)
+  description = "A map specifying how many instances of each type should be created in the operations subnet (e.g. { \"kali\": 1 })."
```
Comment (jsf9k) [inline variables.tf]: See previous comment about adding a list of supported keys.

**Review by jsf9k — CHANGES_REQUESTED** [review]
I requested some minor changes.  Great documentation!

File: README.md (hunk 1 above)
Comment (dav3r) [reply README.md]: Great suggestion, thanks!  Added in https://github.com/cisagov/cool-assessment-terraform/pull/16/commits/3e2cf9971b31be012a77dc0d310486418685af3f.

File: variables.tf (hunk 2 above)
Comment (dav3r) [reply variables.tf]: Added in https://github.com/cisagov/cool-assessment-terraform/pull/16/commits/3e2cf9971b31be012a77dc0d310486418685af3f.

**Review by jsf9k — APPROVED** [review]

**Review by mcdonnnj — APPROVED** [review]
Strong work 💪

**Review by felddy — APPROVED** [review]
You make this look easy!  🦅

**Pull request merged**